src/HX711.cpp

```diff
@@ -0,0 +1,39 @@
+#include "HX711.h"
+
+// Obtain non-calibrated average weight from HX711
+int HX711::get_raw_weight(int nbr_samples){
+    int raw_weight = 0;
+    int temp = 0;
+
+    // Obtain average weight
+    for (int i = 0; i < nbr_samples; i++){
+      // Trigger HX711 for single weight measurement
+      for (int i = 0; i < 24; i++){
```

Comment:
Add a space between parentheses and brackets like this:
```suggestion
      for (int i = 0; i < 24; i++) {
```